io.edgehog.devicemanager.AgentInfo.json

```diff
@@ -0,0 +1,24 @@
+{
+  "interface_name": "io.edgehog.devicemanager.AgentInfo",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
+  "ownership": "device",
+  "mappings": [
+    {
+      "endpoint": "/name",
+      "type": "string",
+      "description": "Name of the agent. Example value: `edgehog-esp32-device`"
+    },
+    {
+      "endpoint": "/url",
+      "type": "string",
+      "description": "URL to the agent code. Example value: `https://github.com/edgehog-device-manager/edgehog-esp32-device`"
```

Comment:
URL that uniquely identifies the Edgehog agent implementation. Example value: `https://github.com/edgehog-device-manager/edgehog-esp32-device`.

Reply:
s/Edgehog agent/Edgehog runtime/g